packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -658,6 +658,21 @@ void main() {
       skip: skipExpectsWithKnownBugs,
     );
   }, skip: skipTestsWithKnownBugs);
+
+  test('TextPainter - empty text baseline', () {
+    final TextPainter painter = new TextPainter()
+      ..textDirection = TextDirection.ltr;
+    painter.text = const TextSpan(
+      text: '',
+      style: const TextStyle(fontFamily: 'Ahem', fontSize: 100.0, height: 1.0),
+    );
+    painter.layout();
+    expect(
+      // Returns -1
+      painter.computeDistanceToActualBaseline(TextBaseline.alphabetic), 80.0,
+      skip: skipExpectsWithKnownBugs,
+    );
+  }, skip: skipTestsWithKnownBugs);
```

Comment:
with your fix this doesn't actually fail, right?

Reply:
With my "fix" the painter returns 84 in this case. As far as I understand it, it should be 80.

84 is a big improvement over -1, however I think it's still wrong.

Reply:
Ah, right.